spec.md

```diff
@@ -899,6 +899,7 @@ The following example demonstrates instantiating a wire with the given name
 wire mywire: UInt
 ```
 
+
```

Comment:
Stray whitespace change (and not consistently using two spaces before these headers, see `## Invalidates` below).